### PR TITLE
Updated example in README to be more clarifying

### PR DESCRIPTION
**Pull request by slavafomin**
I've updated README to be more explanative. I've tried to mitigate common problems people have when working with Gulp itself (and they have a lot of them, especially with async tasks).

This addresses #19.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,20 +18,25 @@ Allows to create a gulp files in classes, each method of which can be a gulp tas
     
     ```typescript
     import {Gulpclass, Task, SequenceTask} from "gulpclass";
-
-    let gulp = require("gulp");
-    let del = require("del");
+    import * as gulp from "gulp";
+    import * as del from "del";
```
Comment (pleerock) [inline README.md:22]: the reason why I used "require" in example because to use import syntax you have to install gulp and del type definitions

Comment (slavafomin) [reply README.md:22]: What's wrong with installing definitions? Isn't it the reason why we use TypeScript in the first place?

Comment (pleerock) [reply README.md:22]: just not to make first-time typescript users who don't yet know about typings to figure out for those errors (i was in same situation when I first used typescript).

Comment (slavafomin) [reply README.md:22]: Typings are integral part of TypeScript ecosystem, we shouldn't try to hide this or to protect new users from it. I think, the sooner they will encounter and learn this subject — the better for them.

Comment (slavafomin) [reply README.md:22]: If you want to be friendly we can always add a notice about this topic somewhere in the README.

**Pull request merged**